src/components/Cards/CardSquare/CardSquare.scss

```
@@ -46,3 +44,26 @@
   height: 45px;
   margin: 0 auto 30px auto;
 }
+
+@media (max-width: $breakpoint-mobile) {
+  .card-square {
+    margin: 19px 16px;
```

lets make these even numbers divisible by 4 or 8 so
```suggestion
    margin: 20px 16px;
```
or just use 16px